src/functions/services/clientCredentialsService/clientCredentialsService.test.ts

```diff
@@ -61,16 +72,46 @@ describe("Client Credentials Service", () => {
       });
     });
 
+    describe("redirect_uri validation", () => {
+      describe("Given redirect_uri is not present", () => {
+        it("Returns a log", () => {
+          mockStoredClientCredentials.redirect_uri = "";
```

Comment:
This can be removed now that the interface has been updated to have this field as an optional parameter